### PR TITLE
Solving Issue #943 with a Tip note

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added a tip note explaining the answer to this issue (#943)

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 1]
```diff
@@ -163,6 +163,14 @@ The graphic below describes the actual end state:
 
 ![State before decommissioning Exchange servers](media/c692f0af-6536-4bc9-950d-58a1e486525f.jpg)
 
+>[!TIP]
+> In case that you decide to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud.
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
> If you choose to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. Each service will authenticate users independently: 
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 2]
```diff
@@ -163,6 +163,14 @@ The graphic below describes the actual end state:
 
 ![State before decommissioning Exchange servers](media/c692f0af-6536-4bc9-950d-58a1e486525f.jpg)
 
+>[!TIP]
+> In case that you decide to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. 
+> So each service will authenticate users independenly:
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 3]
```diff
@@ -163,6 +163,14 @@ The graphic below describes the actual end state:
 
 ![State before decommissioning Exchange servers](media/c692f0af-6536-4bc9-950d-58a1e486525f.jpg)
 
+>[!TIP]
+> In case that you decide to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. 
+> So each service will authenticate users independenly: 
+> * **O365 identity service** will manage online request
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
> * **Office 365 identity services** will manage online requests
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 4]
```diff
@@ -163,6 +163,14 @@ The graphic below describes the actual end state:
 
 ![State before decommissioning Exchange servers](media/c692f0af-6536-4bc9-950d-58a1e486525f.jpg)
 
+>[!TIP]
+> In case that you decide to remove ADFS from your infrastructure, Azure AD Connect will synchronize your on-premises credentials with the cloud. 
+> So each service will authenticate users independenly: 
+> * **O365 identity service** will manage online request 
+> * **Active directory** will manage the internal authentication
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
> * **Azure AD** will manage internal authentication
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 5]
```diff
@@ -163,6 +163,14 @@ The graphic below describes the actual end state:
...
+> 
+>  If you don't have any mailbox on-premises, you can safely decommission your exchange server(s).
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
>  If you don't have any on-premises mailbox(es), you can safely decommission your exchange server(s).
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 6]
```diff
@@ -90,7 +90,7 @@ Since the hybrid customer base is very diverse, trying to fit all of them into "
 
 3. Next, you should remove the Service Connection Point (SCP) values on your Exchange servers. This ensures that no SCP's are returned, and the client will instead use the DNS method for Autodiscover. An example is shown below:
 
-   ```
+   ```Powershell
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
   ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 7]
```diff
@@ -129,7 +129,7 @@ Since the hybrid customer base is very diverse, trying to fit all of them into "
 
    2. Run the following command:
 
-      ```
+      ```Powershell
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
      ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 8]
```diff
@@ -139,7 +139,7 @@ Since the hybrid customer base is very diverse, trying to fit all of them into "
...
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
      ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 9]
```diff
@@ -177,7 +185,7 @@ The graphic below describes the actual end state:
 
 3. Next, you should remove the Service Connection Point (SCP) values on your Exchange servers. This ensures that no SCP's are returned, and the client will instead use the DNS method for Autodiscover. An example is shown below:
 
-   ```
+   ```Powershell
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
   ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 10]
```diff
@@ -186,7 +194,7 @@ The graphic below describes the actual end state:
 
 4. To prevent the hybrid configuration objects from being recreated in the future, you should remove the hybrid configuration object from Active Directory. To do this, open the Exchange Management Shell and run the following:
 
-   ```
+   ```Powershell
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
   ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 11]
```diff
@@ -200,7 +208,7 @@ The graphic below describes the actual end state:
 
    2. Run the following command:
 
-      ```
+      ```Powershell
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
      ```PowerShell
```

File: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md [hunk 12]
```diff
@@ -210,7 +218,7 @@ The graphic below describes the actual end state:
...
```
Comment (nenonix) [inline Exchange/ExchangeHybrid/decommission-on-premises-exchange.md]:
```suggestion
      ```PowerShell
```

**Pull request merged**